platform/jobs/edxTravisCounts.groovy

```diff
@@ -5,53 +5,38 @@ Binding bindings = getBinding()
 config.putAll(bindings.getVariables())
 PrintStream out = config['out']
 
-// Travis Jobs
-
-// The travis-counts job
 Map prJob = [ name: 'travis-counts',
               deprecated: false,
               worker: 'coverage-worker',
               branch: '*/master'
               ]
 
-/* Iterate over the job configurations */
-secretMap.each { jobConfigs ->
-
-    Map jobConfig = jobConfigs.getValue()
-
-    /* Test secret contains all necessary keys for this job */
-    assert jobConfig.containsKey('open')
-    assert jobConfig.containsKey('jobName')
-    assert jobConfig.containsKey('url')
-    assert jobConfig.containsKey('subsetJob')
-
-    job(jobConfig['jobName']) {
 
-        logRotator {
-            daysToKeep(5)
-            numToKeep(201)
-        }
-        concurrentBuild(true)
-        label('coverage-worker')
-        environmentVariables {
-            env('SUBSET_JOB', jobConfig['subsetJob'])
-        }
-        scm {
-            multijob {
-                remote {
-                    url('https://github.com/edx/edx-e2e-tests.git')
-                    refspec(jobConfig.refspec)
-                    }
-                }
-                browser()
-                branch('\*/master')
-                    cleanBeforeCheckout()
-                }
+job(jobConfig['jobName']) {
+
+    logRotator {
+        daysToKeep(5)
+        numToKeep(201)
+    }
+    concurrentBuild(true)
+    label('coverage-worker')
+    environmentVariables {
```

Comment:
The "SUBSET_JOB' environment variable is only used when we run a jobs tasks on separate shads, as we do with python, lettuce and bok choy tests. We can get rid of this as well.